index.htm

```diff
@@ -263,6 +265,50 @@ <h3 class="pagesListHeader" id="thrCat"><strong><i class="fas fa-trophy"></i>&en
         </div>
     </div>
 
+      <div class="jumbotron pagesMainBody">
```

Comment:
Add the form where the current `Join us!` button is since this is essentially replacing that.